packages/react-native/React/Views/RCTView.h

```diff
@@ -135,4 +135,6 @@ extern const UIAccessibilityTraits SwitchAccessibilityTrait;
 @property (nonatomic, assign) RCTBubblingEventBlock onGotPointerCapture;
 @property (nonatomic, assign) RCTBubblingEventBlock onLostPointerCapture;
 
+@property (nonatomic, strong) id transformProp;
+@property (nonatomic, copy) NSString* transformOriginProp;
```

Comment:
Not really convention to suffix these with `prop` (everything here is a prop). What about `rawTransform` (preferably with a better type than `id`) and `transformOrigin`